addons/account/models/account_move.py

```diff
@@ -556,8 +576,7 @@ def _apply_cash_rounding(self, diff_balance, diff_amount_currency, cash_rounding
             :return:                        The newly created rounding line.
             '''
             rounding_line_vals = {
-                'debit': diff_balance > 0.0 and diff_balance or 0.0,
-                'credit': diff_balance < 0.0 and -diff_balance or 0.0,
+                **self._convert_to_debit_credit(diff_balance),
```

Comment:
au secours

Reply:
As explained before, the only way to do it properly is to pass `balance` directly.